src/main/scala/akka/persistence/cassandra/compaction/SizeTieredCompactionStrategy.scala

```diff
@@ -15,12 +15,11 @@ class SizeTieredCompactionStrategy(config: Config) extends BaseCompactionStrateg
       .asScala
       .map(_.getKey)
       .forall(SizeTieredCompactionStrategy.propertyKeys.contains(_)),
-    s"Config contains properties not supported by a ${SizeTieredCompactionStrategy.ClassName}"
-  )
+    s"Config contains properties not supported by a ${SizeTieredCompactionStrategy.ClassName}")
 
   val bucketHigh: Double = if (config.hasPath("bucket_high")) config.getDouble("bucket_high") else 1.5
   val bucketLow: Double = if (config.hasPath("bucket_low")) config.getDouble("bucket_low") else 0.5
-  val coldReadsToOmit: Double = if (config.hasPath("cold_reads_to_omit")) config.getDouble("cold_reads_to_omit") else 0.05
+  // FIXME val coldReadsToOmit: Double = if (config.hasPath("cold_reads_to_omit")) config.getDouble("cold_reads_to_omit") else 0.05
```

Comment:
I get something similar to https://issues.apache.org/jira/browse/CASSANDRA-9203 with the `cold_reads_to_omit` setting. I think it was added in https://github.com/krasserm/akka-persistence-cassandra/pull/110, so do you know anything about that setting for Cassandra 3.0, @jparkie ?